webui/react/src/shared/utils/datetime.ts

```diff
@@ -1,12 +1,15 @@
 import dayjs from 'dayjs';
 import utc from 'dayjs/plugin/utc';
 
+// TODO: check types for this.
+// @ts-ignore
```

Comment:
chore: just adding as a reminder if you wanted to address this before PR wrapup

Reply:
This was changed in a future commit